signingscript/docker.d/passwords.yml

```diff
@@ -144,6 +144,12 @@ in:
              {"$eval": "AUTOGRAPH_AUTHENTICODE_PASSWORD"},
              ["autograph_authenticode", "autograph_authenticode_stub"]
             ]
+          - ["https://autograph-external.stage.autograph.services.mozaws.net",
```

Comment:
We should use Production Autograph for dep/dev unless we're specifically testing the Staging Autograph instance.
Essentially, the Autograph team reserves the right to bring the Staging Autograph instance down for hours or days at a time if needed, but we need PRs, CI signing tasks, and staging releases to work more reliably than that. Therefore we have a "fake", "dep", self-signed key on the production cluster.